streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryTimeOrderedKeyValueBuffer.java

```diff
@@ -74,12 +74,12 @@ public Builder(final String storeName) {
 
         @Override
         public StoreBuilder<StateStore> withCachingEnabled() {
-            throw new UnsupportedOperationException();
```

Comment:
We should add JavaDoc and point out that he method can be called, but won't do anything, as caching is not supported. Similar below.

Reply:
done.